libs/template/themes/src/fes.theme.ts

```diff
@@ -2,6 +2,7 @@ import { Theme } from '@spryker-oryx/experience';
 import { materialDesignIcons } from '@spryker-oryx/resources';
 import { defaultBreakpoints } from '@spryker-oryx/themes/breakpoints';
 
+/** @deprecated fes theme will be deprecated in upcoming major versions */
```

Comment:
Why you don't use jsDoc. Also you can check the message in the already created code. Alsp suggest what user can use instead

/**
 * @deprecated Since version `which one` Use backofficeTheme instead.
*/